typescript/infra/src/agents/key-utils.ts

```diff
@@ -41,14 +37,17 @@ export const kathyAddresses: LocalRoleAddresses =
 export const awMultisigAddresses: ChainMap<{ validators: Address[] }> =
   localAWMultisigAddresses as ChainMap<{ validators: Address[] }>;
 
-const debugLog = debug('infra:agents:key:utils');
+const debugLog = rootLogger.child({ module: 'infra:agents:key:utils' }).debug;
 
 export interface KeyAsAddress {
   identifier: string;
   address: string;
 }
 
-const CONFIG_DIRECTORY_PATH = path.join(__dirname, '../../config');
+const CONFIG_DIRECTORY_PATH = path.join(
+  path.dirname(fileURLToPath(import.meta.url)),
```

Comment:
what a mouthful
is this the new idiom?

Reply:
Sadly yes because they wanted a standard that could work for node, browser, and worker environments. I can tighten it up a little bit with deconstructed path imports.